content/master/concepts/pods.md

```diff
@@ -0,0 +1,384 @@
+## RBAC manager pod
+<!-- vale Microsoft.HeadingAcronyms = YES -->
+The Crossplane RBAC manager pod automates required Kubernetes RBAC permissions
+for Crossplane and Crossplane Providers. 
+
+{{<hint "note" >}}
+Crossplane installs and enables the RBAC manager by default.
+Disabling the RBAC manager requires manual Kubernetes permissions definitions
+for proper Crossplane operations. 
+
+The 
+[RBAC manager design document](https://github.com/crossplane/crossplane/blob/master/design/design-doc-rbac-manager.md) 
+provides more comprehensive details on the Crossplane RBAC requirements.
+{{< /hint >}}
+
+### Disable the RBAC manager 
+
+Disable the RBAC manager after installation by deleting the
+`crossplane-rbac-manager` deployment from the `crossplane-system` namespace.
+
+Disable the RBAC manager before installation by editing the Helm `values.yaml`
+file, setting `rbacManager.deploy` to `false`.
+
+{{< hint "note" >}}
+
+Instructions for changing Crossplane pod settings during installation are in the
+[Crossplane Install]({{<ref "../software/install">}}) section. 
+{{< /hint >}}
+
+<!-- vale Microsoft.HeadingAcronyms = NO -->
+<!-- allow 'RBAC' since that's the name -->
+### RBAC init container
+<!-- vale Microsoft.HeadingAcronyms = YES -->
+
+The RBAC manager requires the `CompositeResourceDefinition` and
+`ProviderRevision` resources to be available before starting.
+
+The RBAC manager init container waits for these resources before starting the 
+main RBAC manager container. 
+
+### RBAC manager container
+
+The RBAC manager container preforms the following tasks:
+* creating and binding RBAC roles to Provider ServiceAccounts, allowing 
+  them to control their managed resources
+* allowing the `crossplane` ServiceAccount to create managed resources
+* creating ClusterRoles to access Crossplane resources in all namespaces
+* creating Roles to access Crossplane resources in specific namespaces
+
+Use the [ClusterRoles]({{<ref "#crossplane-clusterroles">}}) to grant access to all Crossplane resources in the
+cluster.  
+Use the [Roles]({{<ref "#crossplane-roles" >}}) to only grant access to Crossplane Claims. 
+
+#### Crossplane ClusterRoles
+
+The RBAC manager creates four Kubernetes ClusterRoles. These Roles grant 
+permissions over cluster wide Crossplane resources. 
+
+<!-- vale Google.Headings = NO -->
+<!-- disable heading checking for the role names -->
+<!-- vale Google.WordList = NO -->
+<!-- allow "admin" -->
+##### crossplane-admin
+<!-- vale Google.WordList = YES -->
+
+The `crossplane-admin` ClusterRole has the following permissions:
+  * full access to all Crossplane types
+  * full access to all secrets and namespaces (even those unrelated to Crossplane)
+  * read-only access to all cluster RBAC roles, CustomResourceDefinitions and
+    events
+  * ability to bind RBAC roles to other entities. 
+
+View the full RBAC policy with 
+
+```shell
+kubectl describe clusterrole crossplane:aggregate-to-admin
```

Comment:
```suggestion
kubectl describe clusterrole crossplane-admin
```

`crossplane:aggregate-to-admin` isn't the full role, it's just part of it. `crossplane-admin` is the full role.

(Ditto for the edit, view, and browse roles below.)